src/include/utils.hh

```diff
@@ -223,3 +223,7 @@ static inline uint32_t hash_block(const void *buf, size_t len)
 }
 
 uint32_t hash_file(const std::string &filename);
+
+// Searches for an executable named file in the directories named by the PATH
+// environment variable.
+int look_path(const std::string &file, std::string *out);
```

Comment:
You don't have to do that here and now, but I think it's high time to ditch c++03 in kcov (and the MacOS port already depends on newer), and then this would be better to write as

`std::optional<std::string> look_path(...)`

Reply:
Looking at CMake, for building `kcov` you use `--std=c++17`, for tests `--std=c++0x` and, only once, `--std=c++1y` (that is deprecated after C++14).

Reply:
Yes, the C++1y was used for the unit tests, which are sort of stale now anyway. I moved to c++17 after som MacOS rewrites, but forgot that it was done everywhere and not just for the Mac build (which is probably all code that needs it).

Anyway, you don't have to make that change here, I think it's fine as it is.